runtime/rasdump/javadump.cpp

```diff
@@ -2021,7 +2021,7 @@ JavaCoreDumpWriter::writeHookInfo(struct OMRHookInfo4Dump *hookInfo)
 	_OutputStream.writeCharacters(timeStamp);
 	_OutputStream.writeInteger64(hookInfo->startTime % 1000, ".%03llu");
 	_OutputStream.writeCharacters("\n");
-	_OutputStream.writeCharacters("4HKDURATION           DurationMs: ");
+	_OutputStream.writeCharacters("4HKDURATION           Durationus: ");
```

Comment:
I would prefer the presentation adopted below for total time (with 'us' after the value).

Reply:
Agreed. The addition has been made

Reply:
Please remove the (now) redundant 'us' suffix from 'Durationus', but keep the length of that string the same by adding two more spaces after the colon(`:`).

Reply:
I had asked that you put those extra spaces *after* the colon, not before, but I won't hold this up for that.